android/capacitor/src/main/java/com/getcapacitor/CapConfig.java

```diff
@@ -301,6 +301,13 @@ private boolean validateScheme(String scheme) {
             return false;
         }
 
+        // Non-http(s) schemes are not allowed to modify the URL path as of Android Webview 117
```

Comment:
I added this warning to be logged to make this hopefully stand out more. Does this make sense? and do we want to warn people each time?